### PR TITLE
Update use-intune-config-manager-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by dlmsft**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/use-intune-config-manager-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ You can use [Microsoft Endpoint Manager](/mem/endpoint-manager-overview) to conf
 
 6. Next to **Configuration settings**, choose **Edit**.
 
+> [!IMPORTANT]
+> AllowOnAccessProtection and AllowIntrusionPreventionSystem Antivirus settings are officially being deprecated and as such cannot be configured.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/use-intune-config-manager-microsoft-defender-antivirus.md]:
```suggestion
> AllowOnAccessProtection and AllowIntrusionPreventionSystem antivirus settings are officially being deprecated and as such cannot be configured. 
```

File: microsoft-365/security/defender-endpoint/use-intune-config-manager-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -45,6 +45,9 @@ You can use [Microsoft Endpoint Manager](/mem/endpoint-manager-overview) to conf
 
 6. Next to **Configuration settings**, choose **Edit**.
 
+> [!IMPORTANT]
+> AllowOnAccessProtection and AllowIntrusionPreventionSystem Antivirus settings are officially being deprecated and as such cannot be configured. 
+
 7. Expand the **Scan** section, and review or edit your scanning settings.
 
 8. Choose **Review + save**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/use-intune-config-manager-microsoft-defender-antivirus.md]:
```suggestion
8. Choose **Review + save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dlmsft Please implement the suggested changes. Thanks

**Review by dlmsft — COMMENTED** [review]
approved all suggested changes

**Comment by JohanFreelancer9** [comment]
@dlmsft - please also commit the second suggested change. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @dlmsft @yogkumgit @JohanFreelancer9

**Pull request merged**